packages/core/src/components/forms/_control-group.scss

```diff
@@ -215,6 +215,35 @@ Styleguide components.forms.control-group
   }
 
   /*
+  Responsive control groups
+
+  Add the class `pt-fill` to a control group to make all controls expand equally to fill the
+  available space. Then add the class `pt-fixed` to individual controls to revert them to their
+  original default sizes.
```

Comment:
As a utility class, `pt-fixed` suggests to me something to do with `position: fixed`. Is there another name that we could use here, or are we okay with that?